processing/src/main/java/org/apache/druid/query/dimension/PredicateFilteredDimensionSelector.java

```diff
@@ -82,7 +86,7 @@ public boolean matches()
           }
         }
         // null should match empty rows in multi-value columns
-        return nullRow && value == null;
+        return nullRow && (includeUnknown || value == null);
```

Comment:
So if value is not null, an empty row MVD can match that value if includeUnknown is set to true?

Reply:
yeah, mvds consider empty rows as nulls, and also any element being null as null